code/modules/projectiles/boxes_magazines/internal/revolver.dm

```diff
@@ -127,3 +127,10 @@
 	ammo_type = /obj/item/ammo_casing/a45lc
 	caliber = list(CALIBER_45LC)
 	max_ammo = 6
+
+/obj/item/ammo_box/magazine/internal/cylinder/rev308
+	name = ".308 Sporting Revolver Cylinder"
+	ammo_type = /obj/item/ammo_casing/a762/sport
+	caliber = list(CALIBER_762)
+	max_ammo = 10
```

Comment:
I'd prefer this to be something like 8 or 7, that way it doesnt directly compete with the full on semiauto rangemaster.

Reply:
Rangemaster fires faster, that was the balance I had in mind personally. I can change it but this thing has a mean fire delay currently, it probably doesn't need any nerf.

Reply:
Fair enough!